src/redux/walletconnect.js

```diff
@@ -146,7 +146,6 @@ export const walletConnectOnSessionRequest = (uri, callback) => async (
   getState
 ) => {
   let timeout = null;
-  getState().appState;
```

Comment:
WTF lol

Reply:
I reviewed this but posting for geeks: https://cloud.skylarbarrera.com/Screen-Shot-2022-04-14-11-24-13.22.png

Reply:
Shame on you @skylarbarrera

Reply:
😂 😂 😂